api/internals/usecase/activity_usecase.go

```diff
@@ -15,10 +15,10 @@ type activityUseCase struct {
 type ActivityUseCase interface {
 	GetActivities(context.Context) ([]domain.Activity, error)
```

Comment:
```suggestion
	GetActivity(context.Context) ([]domain.Activity, error)
```
ここは単数系でも問題なさそうですけどどうですかね？

Reply:
そのほうが他と揃っていいかもしれないですね